src/VisualStudio/Core/Def/Implementation/Remote/RemotableDataJsonRpcEx.cs

```diff
@@ -144,18 +169,5 @@ private async Task WriteMultipleAssetsAsync(ObjectWriter writer, int scopeId, Ch
                 await remotableData.WriteObjectToAsync(writer, cancellationToken).ConfigureAwait(false);
             }
         }
-
-        protected override void Disconnected(JsonRpcDisconnectedEventArgs e)
-        {
-            // we don't expect OOP side to disconnect the connection. 
-            // Host (VS) always initiate or disconnect the connection.
-            if (e.Reason != DisconnectedReason.LocallyDisposed)
```

Comment:
is ths check no longer relevant?

Reply:
We still have it in `RemoteEndPoint.OnDisconnected`.